src/tests/functional/inference_engine/snippets/fma_fusion.cpp

```diff
@@ -0,0 +1,178 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
```

Comment:
We have a PR on major snippets test infrastructure refactoring/development https://github.com/openvinotoolkit/openvino/pull/10605 So it's probably better to add this tests after the PR is merged. Please highlight this point on our sync, let's discuss it.

Reply:
Tests were refactored in order to align with new snippets test infrastructure. Thanks for the suggestion